### ISSUE TITLE
Test if database is open

### ISSUE THREAD

**Issue by piegamesde** [keepassxreboot/keepassxc-browser#594]
## Expected Behavior
I need to know if the database is currently locked or not, without being associated.

## Current Behavior
I can wait until the `database-locked` action is sent, but if the database is already open there is no way to tell and waiting for this will wait until the database gets locked and unlocked again.

Previously, I used the `get-databasehash` command to test for a closed database but it requires being associated now so that doesn't work anymore since a few updates ago.

## Possible Solutions
- Send the answer of `get-databasehash` unencrypted again.
- Create a new message like `database-status` that will tell me this information

**Comment by varjolintu** [comment]
That request doesn't need any association. Changing the public keys is enough so the request can be sent in a encrypted form.

**Comment by piegamesde** [comment]
Yes, you're right, so `get-databasehash` still works as workaround, but I'd think an explicit action for this might still be better.

**Comment by varjolintu** [comment]
https://github.com/keepassxreboot/keepassxc-browser/issues/1888 will solve this.
